docs/tutorials/tutorial-latest-by.md

```diff
@@ -0,0 +1,225 @@
+|`2024-01-01T01:00:00.000Z`|`funny_bunny1`|20|
+|`2024-01-01T02:00:00.000Z`|`funny_bunny1`|5|
+|`2024-01-01T02:00:00.000Z`|`silly_monkey2`|25|
+|`2024-01-01T03:00:00.000Z`|`funny_bunny1`|10|
+
+LATEST_BY() is an aggregation function. While it's very efficient if there are not a lot of update rows matching the dimension (e.g. "user_id"), it does scan all matching rows with the same dimension. This means, for any permutation of dimensions where there are a lot of updates (e.g. the user has played the game a million times), and a lot of the updates are not coming in timely order, Druid will be forced to process all rows matching the user_id to find the row with max timestamp to give you the latest data. 
+
+You can think about this where if updates constitute 1-5% of your data, you'll get good query performance, if updates constitute 50%+ of your data, your queries will be slow.
+
+To mitigate this, you can set up a periodic batch ingestion job that reindexes modified data into a new datasource for direct querying without grouping to reduce for the cost of these kinds of queries by essentially pre-computing the latest value and store them. Though your view of latest data will not be up to date until the next refresh happens.
+ 
+Alternatively, you can perform ingestion-time aggregation using LATEST_BY and append updates with streaming ingestion into a rolled up datasource. Appending into a time chunk adds new segments and does not perfectly roll up data, so rows may be partial rather than complete rollups, and you may have multiple partially rolled up rows. In this case you still need to use GROUP BY query for correct querying of the rolled-up data source, you can tune automatic compaction right to significantly reduce the number of stale rows and improve your performance
+
+## Use delta values and aggregation for updated values
+
+Instead of appending the latest total value in your events, you could log the change in value with each event and use the aggregator you usually use. This method may allow you to avoid a level of aggregation and grouping in your queries.
+
+Typically, for most applications, all you need to do is to send the event data directly to Druid without pre-processing. For example, when you are sending impression count to Druid, don't send the total impression count since yesterday, send just the recent impression count to Druid. And you can aggregate the total in Druid during query. Druid is very fast on adding up a lot of rows, so this might be counterintuitive to people who are familiar with batching or pre-aggregating data.
+
+For example, consider a datasource with a measure column `y` that you aggregate with SUM, grouped by by another dimension `x`. If you want to update the value of `y` for `x` from 3 to 2, then insert -1 for `y`. This way the aggregation SUM(`y`) is correct for any queries grouped by `x`. This may offer a significant performance advantage but the trade off is that the aggregation has to always be a SUM.
```

Comment:
```suggestion
For example, consider a datasource with a measure column `y` that you aggregate with SUM, grouped by another dimension `x`. If you want to update the value of `y` for `x` from 3 to 2, then insert -1 for `y`. This way the aggregation SUM(`y`) is correct for any queries grouped by `x`. This may offer a significant performance advantage but the trade off is that the aggregation has to always be a SUM.
```